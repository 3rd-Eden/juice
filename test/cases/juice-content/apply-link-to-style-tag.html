<html>
<head>
    <link href="./apply-link-to-style-tag.css" rel="stylesheet">
</head>
<body>
    <p>none</p>
</body>
</html>
