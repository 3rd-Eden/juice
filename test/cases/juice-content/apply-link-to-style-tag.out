<html>
<head>
    <style>p {
    width: 200px;
}</style>
</head>
<body>
    <p style="width: 200px;">none</p>
</body>
</html>
